patches/minecraft/net/minecraft/block/Block.java.patch

```diff
@@ -1332,6 +1332,16 @@
 +    public void addInformation(ItemStack stack, EntityPlayer player, List<String> tooltip, boolean advanced)
 +    {
 +    }
++    
++    /**
++     * @param worldIn The IBlockAccess the Block exists in
++     * @param pos The position of the Block
++     * @param entityIn The Entity requesting the SoundType
```

Comment:
Warn that pos could potentially not have the block actually there

I think the entity @param would be better like "The Entity falling on/breaking/stepping on/digging/placing this block, or null if the context has no entity"